tools/cli/pkg/command/options.go

```diff
@@ -18,8 +18,9 @@ const (
 )
 
 const (
-	tableOutput string = "table"
-	jsonOutput  string = "json"
+	tableOutput        string = "table"
+	jsonOutput         string = "json"
+	customcolumnOutput string = "custom-columns"
```

Comment:
I'd just use `custom` instead of `custom-columns`. Perhaps a small description of what `custom` means would be good.

Reply:
Done.